meta/halyard

```diff
@@ -7,8 +7,12 @@ set -euo pipefail
 FLAGS=''
 if [ -n "$DEBUG" ] ; then
     FLAGS="$FLAGS --debug"
+    if [ -n "$PROFILE" ]; then
```

Comment:
Should this be outside the debug block? I can imagine it's possible you'd want profiling without debug, right?